### PR TITLE
webui: improve documentation on certificates, SSL and NODE_EXTRA_TLS_…

### PR DESCRIPTION
**Pull request by maany**
…CERTS environment configuration

- Make a clear distinction between the types of CA bundles required by the WebUI i.e. for x509 client certificate verification and for verification of the host certificates of Rucio Server, Rucio Auth Server
- Give examples on how these certificates can be mounted into the pods

### CODE REVIEW

File: charts/rucio-webui/README.md [hunk 1]
```diff
@@ -97,15 +97,114 @@ data:
...
+There are two types of CA bundles that are used by the WebUI:
+1. For verification of the x509 client certificates, which is done by Apache by setting the `SSLCACertificateFile` or `SSLCACertificatePath` directive in apache configuration.
+2. For verification of the host certificates of the Rucio server, which is done by NodeJS, ultimately by setting the `RUCIO_WEBUI_SERVER_CA_BUNDLE` environment variable in the WebUI container.
+
+
+#### CA bundles for x509 client certificates
+##### Multiple CA files
+For x509 based authentication, if you have multiple pem encoded CA files, you must mount them at the path specified by the `RUCIO_CA_PATH` environment variable.
+
+You must also set the `RUCIO_CA_PATH` environment variable using the `addtionalEnvs` section to point to the directory where the CA files are mounted. For example, to mount the CA files at `/etc/grid-security/certificates/` and set the `RUCIO_CA_PATH` environment variable to `/etc/grid-security/certificates`, you would set the following in the `values.yaml` file:
```
Comment (rdimaio) [inline charts/rucio-webui/README.md:135]:
```suggestion
You must also set the `RUCIO_CA_PATH` environment variable using the `addtionalEnvs` section to point to the directory where the CA files are mounted. For example, to mount the CA files at `/etc/grid-security/certificates/` and set the `RUCIO_CA_PATH` environment variable to `/etc/grid-security/certificates`, you would set the following in the `values.yaml` file:
```
```suggestion
You must also set the `RUCIO_CA_PATH` environment variable using the `additionalEnvs` section to point to the directory where the CA files are mounted. For example, to mount the CA files at `/etc/grid-security/certificates/` and set the `RUCIO_CA_PATH` environment variable to `/etc/grid-security/certificates`, you would set the following in the `values.yaml` file:
```

File: charts/rucio-webui/README.md [hunk 2]
```diff
@@ -97,15 +97,114 @@ data:
...
+    mountPath: /etc/grid-security/certificates/
+    readOnly: true
+    type: DirectoryOrCreate
+
+additionalEnvs:
+  - name: RUCIO_CA_PATH
+    value: "/etc/grid-security/certificates"
+```
+
+The hostPathMounts is just one way to provide the CA files when you have the CA files locally available on your kubernetes nodes. You can also use other methods like `configmaps` or `secrets` to provide the CA files. Please make sure that the CA files are mounted at the path specified by the `RUCIO_CA_PATH` environment variable.
```
Comment (rdimaio) [inline charts/rucio-webui/README.md:149]:
```suggestion
The `hostPathMounts` is just one way to provide the CA files when you have the CA files locally available on your kubernetes nodes. You can also use other methods like `configmaps` or `secrets` to provide the CA files. Please make sure that the CA files are mounted at the path specified by the `RUCIO_CA_PATH` environment variable.
```

File: charts/rucio-webui/README.md [hunk 3]
```diff
@@ -97,15 +97,114 @@ data:
...
+
+##### Single CA file
+
+If you wish to use a single CA file for x509 authentication, you can provide the CA file as a secret in the same namespace as the WebUI pod, you must mount the file at the path specified by the `RUCIO_CA_FILE` environment variable.
+
+You must also set the `RUCIO_CA_FILE` environment variable using the `addtionalEnvs` section to point to the directory where the CA file is mounted. For example, to mount the CA file at `/etc/grid-security/certificates/ca.pem` and set the `RUCIO_CA_FILE` environment variable to `/etc/grid-security/certificates/ca.pem`, you would set the following in the `values.yaml` file:
```
Comment (rdimaio) [inline charts/rucio-webui/README.md:155]:
```suggestion
You must also set the `RUCIO_CA_FILE` environment variable using the `additionalEnvs` section to point to the directory where the CA file is mounted. For example, to mount the CA file at `/etc/grid-security/certificates/ca.pem` and set the `RUCIO_CA_FILE` environment variable to `/etc/grid-security/certificates/ca.pem`, you would set the following in the `values.yaml` file:
```

**Review by rdimaio — CHANGES_REQUESTED** [review]

File: charts/rucio-webui/README.md (hunk 3 above)
Comment (maany) [reply charts/rucio-webui/README.md:155]: Haha, thanks! I need new batteries in my keyboard, it skips too many letters

**Review by rdimaio — APPROVED** [review]

**Pull request merged**